### ISSUE TITLE
Industrial Coke Oven push fluid

### ISSUE THREAD

**Issue by cloakable** [UnwrittenFun/ImmersiveIntegration#36]
Could you make it so the Industrial Coke Oven pushes fluid out to adjacent inventories? I'm currently using fluid conduit from enderio to extract the creosote oil into a tank, but it would be nice to have it automatically output into said tank.

**Comment by Chinthor** [comment]
I agree with this. The fluid can be extracted with other mods, but (correct me if I'm wrong) there is no way to auto-extract oil just using IE. And that seems wrong.

Little aside, maybe think about having the comparator output from the industrial coke oven based on how much oil is in it?

**Comment by jamesbirtles** [comment]
You could use the pump from IE no?

**Comment by Chinthor** [comment]
(_facepalm_) I had tried that, and apparently I was doing something very wrong. Went to try again before answering you and... it works. I'm a derp.
